### ISSUE TITLE
Should have a refresh-token endpoint

### ISSUE THREAD

**Issue by thgreasi** [balena-io/open-balena-api#575]
We should support a way to refresh the user JWT.
Currently the cloud API has the `/user/v1/refresh-token` but this should also be supported by open-balena.

See: https://www.flowdock.com/app/rulemotion/p-opensource/threads/qnGgwmvppSbFItlU5_U9Rk5JtPn

**Comment by PaulAnnekov** [comment]
Faced this strange error today too. It will be nice to see some progress on resolving the issue.
```
BalenaRequestError: Request error: <!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Cannot GET /user/v1/refresh-token</pre>
</body>
</html>
```

**Comment by dcaputo-harmoni** [comment]
I would also be interested to know if there is a plan to resolve this. Causes balena-cli to continuously need to be re-logged in when using openbalena.

**Comment by thgreasi** [comment]
Hi,
The `/user/v1/refresh-token` endpoints are now available in open-balena-api v0.181.0.
